tests/js/assignment/destructuring-heuristic.js

```diff
@@ -0,0 +1,28 @@
+{{
+  const {
+    id,
+    static: isStatic,
+    method: isMethod,
+    methodId,
+    getId,
+    setId,
+  } = privateNamesMap.get(name);
+
+  const {
+    id1,
+    method: isMethod1,
+    methodId1
+  } = privateNamesMap.get(name);
```

Comment:
I think a case like this one should be added to the CHANGELOG.

Reply:
But it's not a change.

Reply:
This PR is a follow-up to #10222 to prevent some unwanted changes.